lib/improver/wind_downscaling.py

```diff
@@ -336,13 +337,16 @@ def calc_roughness_correction(self, hgrid, uold, mask):
         """Function to perform the roughness correction.
 
         Args:
-            hgrid: 3D or 1D np.array (float)
+            hgrid: 3D or 1D np.array (float32)
```

Comment:
In the IMPROVER documentation, i.e. http://improver.readthedocs.io/en/latest/improver.utilities.cube_manipulation.html#module-improver.utilities.cube_manipulation, it works best if the contents of the brackets is an actual python type. I think this should be `np.float32`, rather than `float32`. Similarly for other uses of `float32` within this program.